File: Front_End/CarFixer/src/app/customer/customer.ts
import { Car } from '../car/Car';

export class Customer{
    ID:number;
    Name: string;
    CariType: string;
    CariHesapKodu: string;
    Authorized: string;
    TaxNumber: string;
    TaxRoom: string;
    Phone: string;
    Phone2: string;
    RiskStatus: string;
    Email: string;
    WebSite: string;
    Fax: string;
    Comment: string;
    Address:string;
    Date:string;
    TimeRegiser:string;
    car:Array<Car>= new Array<Car>();
}

File: CarFixer1/Project.Models/Entites/Customer.cs
using System;
using System.CodeDom;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Project.DAL.Entites.Project.Models
{
  public  class Customer:BaseEntity
    {
        public Customer()
        {
            KayıtTarihi = DateTime.Now;
        }
        public string Name { get; set; }
        public string CariType { get; set; }
        public string  CariHesapKodu { get; set; }
        public DateTime KayıtTarihi { get; set; }
        public string Authorized { get; set; }
        public string TaxNumber { get; set; }
        public string TaxRoom { get; set; }
        public string Phone { get; set; }
        public string Phone2 { get; set; }
        public string RiskStatus { get; set; }
        public string Email { get; set; }
        public string WebSite { get; set; }
        public string Fax { get; set; }
        public string Address { get; set; }
        public string Comment { get; set; }

        //Relational Properties are begin
        public virtual List<Car> Cars  { get; set; }

        /*         public DateTime? VadeGunu { get; set; }
         */
    }
}
